monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -332,7 +331,6 @@ class ConfigurePageComponent extends AuthComponent {
       selectedSection: this.state.selectedSection
     })
     formProperties['schema'] = displayedSchema
-    formProperties['fields'] = {DescriptionField: HtmlFieldDescription};
```

Comment:
This isn't needed at all anymore? How does this affect the configuration page?

Reply:
Currently, nothing is passed to this property so it doesn't affect anything.
If a need to render dangerous html will be raised in the future we'll create a better solution.